src/webhook/test_regular_user_validation.py

```diff
@@ -9,27 +9,22 @@
     "osd-sre-admins"
 ]
 
-def create_request(kind, username, groupname):
+def create_request(username, groupname):
     class FakeRequest(object):
         def __init__(self):
             self.json = {
                 "request": {
                     "uid": "testuser",
                     "operation": "UPDATE",
                     "userInfo": {
-                        "username": username
-                    },
-                    "oldObject": {
-                        "metadata": {
-                            "name": groupname
-                        }
+                        "username": username,
+                        "groups": [
+                            groupname
```

Comment:
Really should provide a way to let this contain multiple group names, and tests that do that.